### PR TITLE
plot PIs

### PR DESCRIPTION
**Pull request by danyang123**

### CODE REVIEW

File: @ABLITZER/plotPIs.m [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+%
+% - plotPIs
+%      - ABLITZER method
+%      -arg1: metric type (positional/turning/shock)
+%      -arg2: (number of arguments) which groups to plot (1. exp only; 2. with self-control; 3. with unpaired control)
+%  TODO:
+
+
+function [OLcontrol,OLexp,p] = plotPIs(obj,...ABLITZAER object
+flag1,... % 1: exp only; 2: with self-control; 3: with unpaired control
```
Comment (young24) [inline @ABLITZER/plotPIs.m]:
```suggestion
numGroups,... % 1: exp only; 2: with self-control; 3: with unpaired control
```
If possible, we should use a specific word to represent the meaning of the variable.

File: @ABLITZER/plotPIs.m [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+flag2)    % "positional" or "turning"
```
Comment (young24) [inline @ABLITZER/plotPIs.m]:
```suggestion
metrics)    % "positional" or "turning"
```
same as the above

File: @ABLITZER/plotPIs.m [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+OLexp = output_task(obj,"OLexp");
+OLcontrol = output_task(obj,"OLcontrol");
+PIsOL = [OLexp.PITime_Baseline;OLexp.PITime_Test;OLcontrol.PITime_Baseline;OLcontrol.PITime_Test;...
+          OLexp.PITurn_Baseline;OLexp.PITurn_Test;OLcontrol.PITurn_Baseline;OLcontrol.PITurn_Test];
+
+[MeanOL,SemOL] = calcMeanSem(PIsOL);
```
Comment (young24) [inline @ABLITZER/plotPIs.m]: Usually, one function only does one thing. Better to decouple this function.

**Comment by young24** [comment]
It is great generally, however, minor issues are pointed out.
@danyang123

**Comment by danyang123** [comment]
I have modified these issues, please check it again. Thanks. @young24

**Review by young24 — APPROVED** [review]
Great. Just merge it.

**Pull request merged**